explainer.html

```diff
@@ -236,42 +259,52 @@ <h2>Use Cases and Requirements</h2>
-        <br>Requirement: An RDF Dataset Canonicalization algorithm.
+        <br>Requirement: An RDF Dataset Canonicalization and Hash algorithms.
       </dd>
+
       <dt id="spaceEfficient">Space-efficient verification of the contents of Datasets</dt>
       <dd>
-        If unique identification of RDF Datasets is possible, one can cryptographically hash the information to establish a storage-efficient way to verify that the information has not changed over time. One property of cryptographic digests is that one can verify data integrity. For example, a small device sending an RDF Dataset to a remote storage location can compute a cryptographic digest for later use in verifying that all the data arrived intact and has not been tampered with. <br>Requirement: An RDF Dataset Canonicalization algorithm whose output can be used as input a cryptographic digest function.
+        If unique identification of RDF Datasets is possible, one can cryptographically hash the information to establish a storage-efficient way to verify that the information has not changed over time. One property of cryptographic digests is that one can verify data integrity. For example, a small device sending an RDF Dataset to a remote storage location can compute a cryptographic digest for later use in verifying that all the data arrived intact and has not been tampered with. 
+        <br>Requirement: An RDF Dataset Canonicalization and Hash algorithms.
         <br>(Contributed by <a href="https://alanhkarp.com/">Alan Karp.)</a>
       </dd>
+
       <dt id="secretConfirmation">Secret confirmation of the contents of Datasets</dt>
       <dd>
-        Since a cryptographic digest is a one-way function, and serves as an abbreviation for the entire RDF Dataset, one can use it in places where secrecy is desired. For example, when ensuring that the transaction history on a distributed ledger is the same between two services, two systems could keep track of the list of transactions in their respective ledgers. Canonicalizing and cryptographically hashing the list of transactions should result in the same cryptographic hash without either party needing to share the list of transactions with the other. <br>Requirement: An RDF Dataset Canonicalization algorithm whose cryptographic digest can be shared to establish the unique identity of the RDF Dataset.
+        Since a cryptographic digest is a one-way function, and serves as an abbreviation for the entire RDF Dataset, one can use it in places where secrecy is desired. For example, when ensuring that the transaction history on a distributed ledger is the same between two services, two systems could keep track of the list of transactions in their respective ledgers. Canonicalizing and cryptographically hashing the list of transactions should result in the same cryptographic hash without either party needing to share the list of transactions with the other. <br>Requirement: An RDF Dataset Canonicalization and Hash algorithms.
```

Comment:
```suggestion
        Since a cryptographic digest is a one-way function, and serves as an abbreviation for the entire RDF Dataset, one can use it in places where secrecy is desired. For example, when ensuring that the transaction history on a distributed ledger is the same between two services, two systems could keep track of the list of transactions in their respective ledgers. Canonicalizing and cryptographically hashing the list of transactions should result in the same cryptographic hash without either party needing to share the list of transactions with the other. <br>Requirement: RDF Dataset Canonicalization and Hash algorithms.
```